set.mm

```diff
@@ -662217,9 +662122,7 @@ not even needed (it can be any class).  (Contributed by Glauco
     $( Given an element of the preimage, its function value is in the image.
        (Contributed by Glauco Siliprandi, 2-Jan-2022.) $)
     fnfvima2 $p |- ( ph -> ( F ` B ) e. ( F " C ) ) $=
```

Comment:
This theorem can be removed!?

Reply:
Yes, but the current proof should actually be kept as the main one for fnfvimad (same contributor @glacode).